pom.xml

```diff
@@ -138,6 +138,17 @@
             <artifactId>kafka-connect-storage-common-htrace-core4-shaded</artifactId>
             <version>${confluent.version}</version>
         </dependency>
+        <dependency>
```

Comment:
is this due to a CVE?

Reply:
no, the build is failing with noclassfound exception without it